File: 1071-binary-prefix-divisible-by-5/binary-prefix-divisible-by-5.js
var prefixesDivBy5 = function(nums) {
let output = [];
let curr = 0;
for (let bit of nums){
    curr = (curr*2 + bit) % 5;
    if (curr === 0)
    output.push(true);
    else
    output.push(false);
}    
return output;
};

File: 1071-binary-prefix-divisible-by-5/binary-prefix-divisible-by-5.py
class Solution:
    def prefixesDivBy5(self, nums: List[int]) -> List[bool]:
        output = []
        curr = 0
        for bit in nums:
            curr = (curr * 2 + bit) % 5
            if curr == 0:
                output.append(True)
            else:
                output.append(False)
        return output
        